launcher/config.go

```diff
@@ -403,7 +403,6 @@ func setupMetrics(c Config) (func() error, error) {
 		TemporalityPreference:   c.MetricExporterTemporalityPreference,
 		MetricsBuiltinsEnabled:  c.MetricsBuiltinsEnabled,
 		MetricsBuiltinLibraries: c.MetricsBuiltinLibraries,
-		UseLightstepMetricsSDK:  c.UseLightstepMetricsSDK,
```

Comment:
Since we are no longer using `c.UseLightstepMetricsSDK` can we also cleanup the `WithLightstepMetricsSDK` function https://github.com/lightstep/otel-launcher-go/blob/main/launcher/config.go#L189

Reply:
Also is the `UseLightstepMetricsSDK` needed here or can it be removed along with references in the tests? https://github.com/lightstep/otel-launcher-go/blob/main/launcher/config.go#L238

Reply:
I removed those, thanks!